faq.rst

```diff
@@ -181,8 +183,8 @@ best to run a ``make distclean`` to ensure that all previously built files
 are removed.
 
 
-I want to keep a separate working copy per development branch, is it possible?
-''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+I want to keep a separate working copy per development branch, is it's possible?
```

Comment:
@Mariatta I fixed the typo. Thanks!
@zware If you want to add an entry for it then go for it!